portal/config/config.py

```diff
@@ -111,7 +111,8 @@ class BaseConfig(object):
 
     # Medidata integration configuration
     # disable creation and editing of patients when active
-    MEDIDATA_RAVE_ORG = os.environ.get('MEDIDATA_RAVE_ORG')
+    PROTECTED_ORG = os.environ.get('PROTECTED_ORG')
+    PROTECTED_FIELDS = os.environ.get('PROTECTED_FIELDS')
```

Comment:
the values of environment variables are strings and need to be converted to the correct type (if we need something besides a string), vs `.cfg` files which actually are treated the same as `.py` files

```suggestion
    PROTECTED_FIELDS = os.environ['PROTECTED_FIELDS'].split(',') \
        if os.environ.get('PROTECTED_FIELDS') else None
```